### DISCUSSION TITLE
[Firebase Messaging] Get actions when user taps notification banner

### DISCUSSION THREAD

**Discussion by delfme** [firebase/flutterfire#12232] [Feature request]
We use `getInitialMessage()` and `onMessageOpenedApp` to grab the event "user tapped a notification banner" and then navigate user to a screen.

Is there a way to also grab the button tapped by the user in case that actions are attached to the notifications? We need this to manage Accept/Decline of an icoming phone call.
On IOS, we are using `ios Notification Service Extension`, so we cannot rely on `flutter_local_notification` and we'd need firebase above methods to being able to get actions selected by the user when tapping the notification banner.

**Comment by darshankawar** [comment]
@delfme 
I think this is not supported by the plugin per https://github.com/firebase/flutterfire/issues/1484#issuecomment-618278471.
I am moving this issue to discussion if anybody else has any other opinion or solution.
